diff --git a/node_modules/react-native-intent-launcher/android/build.gradle b/node_modules/react-native-intent-launcher/android/build.gradle
index ae3c73b..f469591 100644
--- a/node_modules/react-native-intent-launcher/android/build.gradle
+++ b/node_modules/react-native-intent-launcher/android/build.gradle
@@ -5,8 +5,8 @@ def safeExtGet(prop, fallback) {
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion safeExtGet("compileSdkVersion", "28.0.2")
-    buildToolsVersion safeExtGet("buildToolsVersion", 28)
+    compileSdk safeExtGet("compileSdkVersion", safeExtGet("compileSdk", 28))
+    buildToolsVersion safeExtGet("buildToolsVersion", "28.0.3")
 
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
@@ -16,7 +16,6 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:+"
-    compile "com.facebook.react:react-native:+"  // From node_modules
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation "com.facebook.react:react-native:+"  // From node_modules
 }
